frontend/src/home/AddPlanPopper.tsx

```diff
@@ -268,6 +279,35 @@ function AddPlanPopperComponent(props: Props) {
     );
   };
 
+  const renderCatalogYearDropdown = () => {
+    let catalogYears = [
+      ...Array.from(new Set(allMajors.map(maj => maj.yearVersion.toString()))),
+    ];
+    return (
+      <Autocomplete
+        style={{ marginTop: "10px", marginBottom: "5px" }}
+        disableListWrap
+        options={catalogYears}
+        renderInput={params => (
+          <TextField
+            {...params}
+            variant="outlined"
+            label="Catalog Year"
+            fullWidth
+          />
+        )}
+        value={selectedCatalogYear ? String(selectedCatalogYear) : ""}
```

Comment:
NIT: I think you can pass it in selectedCatalogYear directly since `value` can be null.